bf_opencore/tests/test_network.py

```
@@ -233,8 +234,8 @@ def test_cidr_bad_json(nwk_authorized_client):
 
 def test_asset_in_network_old_api(nwk_authorized_client):
     """Ensure we can determine network membership."""
-    dummy_asset = bf_mod.Asset.objects.create(ip_address='10.0.0.1')
-    network = bf_mod.Network.objects.create()
+    dummy_asset = models.Asset.objects.create(ip_address='10.0.0.1')
```

Ruff (0.15.1)

[error] Local variable `dummy_asset` is assigned to but never used

Remove assignment to unused variable `dummy_asset`

(F841)